operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 * [ENHANCEMENT] Update the `rollout-operator` subchart to `0.2.0`. #3624
 * [ENHANCEMENT] Add ability to manage PrometheusRule for metamonitoring with Prometheus operator from the Helm chart. The alerts are disabled by default but can be enabled with `prometheusRule.mimirAlerts` set to `true`. To enable the default rules, set `mimirRules` to `true`. #2134 #2609
 * [BUGFIX] Enable `rollout-operator` to use PodSecurityPolicies if necessary
+* [FEATURE] Add cache support for admin bucket. #3740
```

Comment:
we usually order the changelog like

```
- [CHANGE]
- [FEATURE]
- [ENHANCEMENT]
- [BUGFIX]
```

can you please move this 3 lines up so it's at the top?

Also, to be super clear you can mention that this change only applies to GEM
```suggestion
* [FEATURE] Add cache support for GEM's admin bucket. #3740
```

Reply:
Fixed